src/contour/contour.yml

```diff
@@ -176,6 +176,9 @@ profiles:
         # whether or not to put the window into maximized mode.
         maximized: false
 
+        # Determines if BEL character sound is "off" or "default" or is given by file at "path"
```

Comment:
I think we can make this a multi-line comment, specifically describing each value.